windows/eflow_deploy/check_eflow_vm.yml

```diff
@@ -19,8 +19,10 @@
   ansible.builtin.assert:
     that:
       - win_powershell_cmd_output.stdout is defined
-      - "'Common Base Linux Mariner' in win_powershell_cmd_output.stdout"
-    fail_msg: "Not get 'Common Base Linux Mariner' keyword in file '/etc/os-release' in EFLOW virtual machine."
+      - win_powershell_cmd_output.stdout is search('Common Base Linux Mariner|Microsoft Azure Linux')
+    fail_msg: >-
+      Failed to get OS name 'Common Base Linux Mariner' or 'Microsoft Azure Linux'
+      in file '/etc/os-release' in EFLOW virtual machine.
```

Comment:
I suggest printing the value of `win_powershell_cmd_output.stdout` in this fail message for debugging.

Reply:
Added, thanks.